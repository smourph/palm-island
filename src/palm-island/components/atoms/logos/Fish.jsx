import React from 'react';
import fish from '@internal/resources/images/logo-fish.png';
import './Logo.scss';

export const Fish = () => (
  <span className="logo logo-resource logo-fish">
    <img src={fish} alt="fish"/>
  </span>
);

export default Fish;
