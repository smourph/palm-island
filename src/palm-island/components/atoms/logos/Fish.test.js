import React from 'react';
import {render, screen} from '@testing-library/react';
import Fish from './Fish';

it('should render snapshot', () => {
  const {asFragment} = render(<Fish/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render fish', () => {
  render(<Fish/>);

  const icon = screen.getByAltText('fish');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-fish.png');
});
