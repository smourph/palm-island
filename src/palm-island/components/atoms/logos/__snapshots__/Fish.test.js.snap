// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-resource logo-fish"
>
  <img
    alt="fish"
    src="logo-fish.png"
  />
</span>
`;
